tests/mtest/src/main/scala/tests/BaseSuite.scala

```diff
@@ -5,11 +5,19 @@ import scala.meta.internal.metals.JdkSources
 import scala.meta.internal.mtags
 import scala.meta.internal.semver.SemVer
 import scala.util.Properties
+import munit.internal.FutureCompat._
 import munit.Tag
-import munit.TestOptions
+import munit.TestValues
+import scala.concurrent.Future
+import scala.util.Failure
+import scala.util.Success
+import scala.concurrent.ExecutionContextExecutor
 
 class BaseSuite extends munit.FunSuite with Assertions {
 
+  implicit val ec: ExecutionContextExecutor =
+    scala.concurrent.ExecutionContext.global
```

Comment:
you can use `munitExecutionContext` which is in scope

Reply:
I would actually recommend using it inline, so that we don't accidentally use it implicitly anywhere else

Reply:
I can't figure out where to put this inline... Can you give me a hint. 😬

Reply:
Actually it doesn't matter. I'm going to remove "Rerun" anyways....